File: src/main/java/java_code/part1/코틀린_맛보기.java
package java_code.part1;

public class 코틀린_맛보기 {
    static class Person{
        private String name;
        private int age;

        Person(String name, int age){
            this.name = name;
            this.age = age;
        }
    }

    public static void main(String[] args) {
        Person person1 = new Person("영희", 12);
        Person person2 = new Person("철수", 29);

    }

    public void crate(){
        Person person1 = new Person("영희", 12);
        Person person2 = new Person("철수", 29);
    }
}

File: src/main/java/kotlin_code/part1/코틀린_맛보기.kt
package kotlin_code.part1

data class Person(val name: String, val age: Int? = null)

fun main(args: Array<String>){
    val persons = listOf(Person(name = "영희"),
            Person("철수", age= 29))
    val oldest = persons.maxByOrNull { it.age ?: 0 }
    print("나이가 가장 많은 사람: $oldest")
}